src/widget/wtrackproperty.cpp

```diff
@@ -84,6 +84,18 @@ void WTrackProperty::mouseMoveEvent(QMouseEvent *event) {
     }
 }
 
+void WTrackProperty::mousePressEvent(QMouseEvent* event) {
+    if ((event->buttons() & Qt::LeftButton) && m_pCurrentTrack) {
```

Comment:
```suggestion
    if (event->buttons().testFlag(Qt::LeftButton) && m_pCurrentTrack) {
```

Reply:
not necessary anymore with `mouseDoubleClickEvent(QMouseEvent *event)`. thx